tests/math/test_matrix_manipulation.py

```diff
@@ -197,7 +212,6 @@ def test_autograd(self, i, base_matrix, tol):
     def test_torch(self, i, base_matrix, tol):
         """Tests differentiation in torch by computing the Jacobian of
         the expanded matrix with respect to the canonical matrix."""
-        import torch
```

Comment:
leave previous tests as is please

Reply:
will revert them back but it was causing the black formatting to fail, just a heads up